packages/nhost-js/src/clients/functions.ts

```diff
@@ -18,7 +18,7 @@ export class NhostFunctionsClient {
     })
   }
 
-  async call(url: string, data: any, config?: AxiosRequestConfig): Promise<FunctionCallResponse> {
+  async call<T = unknown, D = any>(url: string, data: D, config?: AxiosRequestConfig): Promise<FunctionCallResponse<T>> {
```

Comment:
You did a good job on refactoring the `request` function in `graphql.ts`. 👍🏼 

Could you please apply the same changes to this function too? Then I think we are good to go.